notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -60,11 +60,165 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import seaborn as sns
</pre><p>3 is <code>cool</code></p>
<br/>

 _Reply via <a href='http://127.0.0.1:8000/test-v1/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>